src/platform/firebase-web.js

```diff
@@ -1,2 +1,2 @@
-const {firebase} = window.firebase;
+const firebase = window.firebase ? window.firebase.firebase : null;
```

Comment:
Why do we need to do this? Seems worth a brief comment.

Reply:
Ok. It's done this way because although `window.firebase` may be filled in with configuration and other non-active ingredients, but `window.firebase.firebase` only has a value if the firebase library is linked in. The unfortunate naming is legacy, I call it p2.